include/aspect/postprocess/point_values.h

```diff
@@ -85,8 +85,24 @@ namespace aspect
         void serialize (Archive &ar, const unsigned int version);
 
       private:
-        std::vector<Point<dim> >                                       evaluation_points;
+        /**
+         * Vector of arrays representing the points where the solution is to be evaluated.
+         */
+        std::vector<std_cxx11::array<double,dim> >                     evaluation_points;
+        /**
+         * Vector of Points representing the points where the solution is to be evaluated
+         * that can be used by VectorTools.
+         */
+        std::vector<Point<dim> >                                       evaluation_points_cartesian;
```

Comment:
It seems strange to me that you need to store the data twice. Do you really need both or can't you store it once and convert when needed?